packages/story-editor/src/components/button/index.js

```diff
@@ -18,68 +18,23 @@
  * External dependencies
  */
 import { forwardRef } from 'react';
-import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
+import { Button, BUTTON_VARIANTS } from '@web-stories-wp/design-system';
 import { ArrowDown as ArrowDownIcon, More as MoreIcon } from '../../icons';
 
-const Base = styled.button.attrs(({ isDisabled }) => ({
-  disabled: isDisabled,
-}))`
-  border-width: 1px;
-  border-style: solid;
-  border-radius: 2px;
-  background: transparent;
-  display: block;
-  min-width: ${({ isIcon }) => (isIcon ? 'initial' : '63px')};
-  max-height: 30px;
-  padding: 0 10px;
-  cursor: pointer;
-
-  &:active {
-    outline: none;
-  }
-
-  svg {
-    width: 1em;
-  }
-
-  ${({ disabled }) =>
-    disabled &&
-    `
-		pointer-events: none;
-		opacity: .3;
-	`}
-`;
-
-const StyledButton = styled(Base)`
-  border: none;
-  padding: 0;
-  width: ${({ width }) => width}px;
-  height: ${({ height }) => height}px;
-  min-width: initial;
-  visibility: ${({ isHidden }) => (isHidden ? 'hidden' : 'visible')};
-  color: ${({ theme }) => theme.colors.standard.white};
-
-  svg {
-    width: ${({ width }) => width}px;
-    height: ${({ height }) => height}px;
-    display: block;
-  }
-`;
-
 export const ArrowDown = (props) => (
-  <StyledButton {...props}>
+  <Button {...props} variant={BUTTON_VARIANTS.ICON}>
     <ArrowDownIcon />
-  </StyledButton>
+  </Button>
 );
 
 export const More = forwardRef(function More(props, ref) {
   return (
-    <StyledButton {...props} ref={ref}>
```

Comment:
So this button is only getting used in the `dropDownMenu` it looks like. Inside of `story-editor/src/components/library/panes/media/local/dropDownMenu` you'll see this `More` getting imported and then get some other styles applied: 

```
const MoreButton = styled(More)`
  position: absolute;
  top: 8px;
  right: 8px;
  background: ${({ theme }) => theme.colors.bg.secondary};
  color: ${({ theme }) => theme.colors.fg.primary};
  border-radius: 100%;
`;
``` 

Since this `More` here in `button/index.js` is only passing props around and grabbing the icon, it could probably just get scrapped completely and then in the dropDownMenu where we have `MoreButton` it can just be `styled(Button).attrs({ variant: BUTTON_VARIANTS.CIRCLE })`. 

Additionally, I'm curious if we can get rid of any of the extra styles on the `MoreButton`. We should be able to apply a `BUTTON_TYPE` to remove the need for the custom colors and the border radius should taken care of by using `CIRCLE` instead of `ICON` (which is just a safeguard for when we need an icon to do something like a button should but it's not styled like one. In this case, it's actually styled as a button so we're fine. 

Lastly, applying `BUTTON_SIZES.SMALL` as the `size`. So it'll end up looking like this:  `.attrs({ variant: BUTTON_VARIANTS.CIRCLE, size: BUTTON_SIZES.SMALL, type: BUTTON_TYPES.SECONDARY (i think) })`.

And that means the `MoreButton` should only need to do position, top, right itself.

Reply:
To just reiterate a little bit of the above, you shouldn't need this at all anymore and can delete lines 34 - 40 for this `MoreButton` because now in `local/dropDownMenu` you're importing `Button` from the design system - so this isn't ever getting used. 

Does that make sense?

Reply:
Sounds good. Will delete it.